content/articles/microsoft-ssis/index.md

```diff
@@ -0,0 +1,185 @@
+- [SSIS Best Practices](#ssis-best-practices)
+- [Conclusion](#conclusion)
+- [Further Reading](#further-reading)
+
+### Key Takeaways
+After reading this article, the reader should be able to understand the following:
+* SSIS components, tasks, and packages
+* How SSIS works
+* Advantages and disadvantages of using SSIS
+* SSIS best practices
+
+### SSIS Components
+1. Control Flow
+The control flow is the main component of the SSIS package. It acts as the SSIS's brain.  It assists in arranging the execution sequence for all of its components. Containers and tasks are controlled using priority restrictions inside the components.
+2. Precedence Constraints
+A precedence constraint is an SSIS component that instructs tasks to run in a specific sequence. Thus it describes the SSIS package's overall process. It directs the implementation of the two related activities by performing the target tasks according to the outcome of the previous task. 
+3. Tasks
+A 'Task' is defined as a discrete piece of activity in an SSIS package. It's similar to a programming language's method or function. Nonetheless, coding approaches are not used in SSIS. Rather, you'll create and customize surfaces using the drag-and-drop method.
+4. Containers
+The container is a component that organizes tasks into workgroups. It additionally enables you to specify variables and events controllers that should be in the range of that particular container, in addition to providing visual continuity.
+In SSIS, there are four different kinds of containers:
+  * A container for sequences.  A Sequence Container enables a user to gather and arrange secondary tasks perform operations, and allocate logs to the container.
+  * A Container for For Loops. A loop container offers a similar capability as the sequence Container, but it additionally allows you to repeat the operations. It is, therefore, predicated on a condition of assessment, such as a looping from One to one hundred.
+  * Container for the Foreach Loop.  A Loop Container also permits looping. However, rather than utilizing a conditional expression, the loop is performed across a group of objects, such as files inside a folder.
+5. Data Flow
+The SSIS tool is mainly used to extort data from the server's storage, alter, and write it to a different location. While Control Flow is the main component of SSIS, Data Flow is its heart.
+6. SSIS Packages
+The 'package' is another vital aspect of SSIS. It comprises a set of tasks that run in a logical order. Primary restrictions are used to control the task's execution sequence. A package can assist a user in saving data to package catalog database on SQL Server. It can store as a .dtsx file, an organized format comparable to Reporting Services'.rdl files.
+7. Parameters
+With minor differences, parameters act similarly to variables. It's simple to take it out of the packaging. It can be specified as a set of values that are supplied before the package can begin.
+
+## SSIS Tasks
```

Comment:
change it to level three header. Look at other published articles to see how headers are formatted.